CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,4 @@
+[cygnus-ngsi] [PostgisSink] Fix PostgreSQL encoding when enabled.
+[cygnus-ngsi] [MysqlSink, PostgisSink, PosgreSQLSink] Update sinks to initialize persistance backend objects without database. (#2004)
+[cygnus-common] Remove database initialization on constructors. (#2004)
```

Comment:
This PR has no any impact in documentation?

Just to check...

Reply:
It's just a bugfix, do not introduce a new behavior

Reply:
NTC